doc/package.nix

```diff
@@ -32,16 +32,24 @@ let
     sed -i -e '/\*Declared by:\*/,+1d' $out
   '';
 in
-pkgs.stdenv.mkDerivation {
-  name = "wire-docs";
-  buildInputs = with pkgs; [
-    mdbook
-    mdbook-alerts
+pkgs.stdenv.mkDerivation (finalAttrs: {
+  pname = "wire-docs";
+  version = "3";
+  nativeBuildInputs = with pkgs; [
+    wire
+    nodejs
+    pnpm.configHook
```

Comment:
I would suggest getting rid of the `with pkgs;` and `pkgs` up top and get each dependency there instead as you're using `callPackage` for this file.